tests/components/stream/test_hls.py

```diff
@@ -70,23 +70,24 @@ def make_segment(segment, discontinuity=False):
                 + "Z",
             ]
         )
-    response.extend(["#EXTINF:10.0000,", f"./segment/{segment}.m4s"]),
+    response.extend(["#EXTINF:10.0000,", f"./segment/{segment}.m4s"])
     return "\n".join(response)
 
 
-def make_playlist(sequence, discontinuity_sequence=0, segments=[]):
+def make_playlist(sequence, segments, discontinuity_sequence=0):
     """Create a an hls playlist response for tests to assert on."""
     response = [
         "#EXTM3U",
         "#EXT-X-VERSION:6",
-        "#EXT-X-TARGETDURATION:10",
+        "#EXT-X-INDEPENDENT-SEGMENTS",
         '#EXT-X-MAP:URI="init.mp4"',
+        "#EXT-X-TARGETDURATION:10",
         f"#EXT-X-MEDIA-SEQUENCE:{sequence}",
         f"#EXT-X-DISCONTINUITY-SEQUENCE:{discontinuity_sequence}",
         "#EXT-X-PROGRAM-DATE-TIME:"
         + FAKE_TIME.strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3]
         + "Z",
-        f"#EXT-X-START:TIME-OFFSET=-{1.5*SEGMENT_DURATION:.3f},PRECISE=YES",
+        f"#EXT-X-START:TIME-OFFSET=-{1.5*SEGMENT_DURATION:.3f}",
```

Comment:
PRECISE=YES was removed. Without PRECISE specified (or with PRECISE=NO), players start from the beginning of the segment where the time offset is found. This makes more sense for now since we are only delivering whole segments.